<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QChartView" name="magnitudeGraphicView">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>21</y>
     <width>431</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QChartView" name="phaseGraphicView">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>451</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>449</width>
     <height>146</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Frequency Min (Hz):</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="3">
     <widget class="QLineEdit" name="denominatorLineEdit"/>
    </item>
    <item row="0" column="2" colspan="3">
     <widget class="QLineEdit" name="numeratorLineEdit"/>
    </item>
    <item row="4" column="0" colspan="5">
     <widget class="QPushButton" name="calculatePushButton">
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLineEdit" name="frequencyMaxlineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Denominator D(s):</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string> Numerator N(s):</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Sigma</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QLineEdit" name="sigmaLineEdit"/>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLineEdit" name="frequencyMinlineEdit"/>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Frequency Max(Hz):</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
